packages/app/src/app/overmind/effects/zeit.ts

```diff
@@ -91,12 +91,16 @@ export default (() => {
         headers: getDefaultHeaders(),
       });
 
-      return response.data;
+      return response.data.user;
     },
     async deleteDeployment(id: string) {
-      return axios.delete(`https://api.zeit.co/v9/now/deployments/${id}`, {
-        headers: getDefaultHeaders(),
-      });
+      const response = await axios.delete(
+        `https://api.zeit.co/v9/now/deployments/${id}`,
+        {
+          headers: getDefaultHeaders(),
+        }
+      );
+      return response.data.deployments;
```

Comment:
I believe this now returns `any`. Should either:

```ts
const response = await axios.delete<{ deployments: Deployment[] }>(...)
```

Sorry, should have stated it in previous review ;-) You could also:

```ts
async deleteDeployment(id: string): Promise<Deployment[]> {
```